spec/features/cdr/cdr_history/cdr_filter_spec.rb

```diff
@@ -0,0 +1,48 @@
+RSpec.describe 'Cdrs index page', type: :feature, js: true do
```

Comment:
when spec file is in `features` folder you don't need to set `type: :feature` explicitly

rspec config `config.infer_spec_type_from_file_location!` will do it for you